doc/api/stream.md

```diff
@@ -2624,6 +2627,30 @@ unless `emitClose` is set in false.
 Once `destroy()` has been called, any further calls will be a no-op and no
 further errors except from `_destroy()` may be emitted as `'error'`.
 
+#### `stream.duplexPair([options])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `options` {Object} A value to pass to both [`Duplex`][] constructors,
+  to set options such as buffering.
+* Returns: {Array} of two [`Duplex`][] instances.
+
+The utility function `duplexPair` returns an Array with two items,
+each being a `Duplex` stream connected to the other side:
+
+```js
+const [ clientSide, serverSide ] = duplexPair();
```

Comment:
nit: perhaps something like this, so it less implies that there is anything special about the order?
```suggestion
const [ sideA, sideB ] = duplexPair();
```